common/src/main/resources/assets/wynntils/lang/zh_tw.json

```diff
@@ -1,982 +1,984 @@
+  "feature.wynntils.containerSearch.filterInGuildMemberList.name": "公會成員搜尋框",
+  "feature.wynntils.containerSearch.highlightColor.description": "醒目標示圈應該是什麼顏色?",
+  "feature.wynntils.containerSearch.highlightColor.name": "醒目標示顏色",
+  "feature.wynntils.containerSearch.name": "容器搜尋醒目標示",
+  "feature.wynntils.cosmeticsPreview.name": "自訂物玩家預覽",
+  "feature.wynntils.customBarsOverlay.name": "自訂義量條",
   "feature.wynntils.customBarsOverlay.overlay.awakenedProgressBar.name": "覺醒條",
-  "feature.wynntils.customBarsOverlay.overlay.baseBar.flip.description": "应该翻轉條的填充方向吗？",
-  "feature.wynntils.customBarsOverlay.overlay.baseBar.flip.name": "翻轉條",
-  "feature.wynntils.customBarsOverlay.overlay.baseBar.shouldDisplayOriginal.description": "是否把被條顯示信息的原來條藏嗎？ ",
-  "feature.wynntils.customBarsOverlay.overlay.baseBar.shouldDisplayOriginal.name": "顯示原來的條？",
-  "feature.wynntils.customBarsOverlay.overlay.baseBar.textColor.description": "文本應該哪個顔色顯示？",
-  "feature.wynntils.customBarsOverlay.overlay.baseBar.textColor.name": "文本色",
-  "feature.wynntils.customBarsOverlay.overlay.baseBar.textShadow.description": "文本陰影要如何顯示？",
-  "feature.wynntils.customBarsOverlay.overlay.baseBar.textShadow.name": "文本陰影",
+  "feature.wynntils.customBarsOverlay.overlay.baseBar.flip.description": "量條的填充方向是否應該反轉?",
+  "feature.wynntils.customBarsOverlay.overlay.baseBar.flip.name": "反轉",
+  "feature.wynntils.customBarsOverlay.overlay.baseBar.shouldDisplayOriginal.description": "是否應該顯示資料的原始形式 (操作條/Boss 血條)? 這個設定只在啟用合適的自訂義量條時可用。",
+  "feature.wynntils.customBarsOverlay.overlay.baseBar.shouldDisplayOriginal.name": "顯示原始量條資料",
+  "feature.wynntils.customBarsOverlay.overlay.baseBar.textColor.description": "文字應該是什麼顏色?",
+  "feature.wynntils.customBarsOverlay.overlay.baseBar.textColor.name": "文字顏色",
+  "feature.wynntils.customBarsOverlay.overlay.baseBar.textShadow.description": "文字陰影應該長什麼樣子?",
+  "feature.wynntils.customBarsOverlay.overlay.baseBar.textShadow.name": "文字陰影",
   "feature.wynntils.customBarsOverlay.overlay.bloodPoolBar.name": "血池條",
   "feature.wynntils.customBarsOverlay.overlay.corruptedBar.name": "腐化條",
-  "feature.wynntils.customBarsOverlay.overlay.focusBar.name": "专注條",
-  "feature.wynntils.customBarsOverlay.overlay.healthBar.healthTexture.description": "血量條使用哪樣的材質顯示？",
+  "feature.wynntils.customBarsOverlay.overlay.focusBar.name": "專注條",
+  "feature.wynntils.customBarsOverlay.overlay.healthBar.healthTexture.description": "血量條的材質應該用什麼樣子的?",
   "feature.wynntils.customBarsOverlay.overlay.healthBar.healthTexture.name": "血量條材質",
   "feature.wynntils.customBarsOverlay.overlay.healthBar.name": "血量條",
-  "feature.wynntils.customBarsOverlay.overlay.manaBankBar.name": "魔力池條",
-  "feature.wynntils.customBarsOverlay.overlay.manaBar.manaTexture.description": "魔力條使用哪樣的材質顯示？",
-  "feature.wynntils.customBarsOverlay.overlay.manaBar.manaTexture.name": "魔力材質",
+  "feature.wynntils.customBarsOverlay.overlay.manaBankBar.name": "魔量條",
```

Comment:
This might be better as showing that its an additional storage of mana beyond the normal mana bar, currently its a bit similar to `mana bar`'s translation

Reply:
Oh, so its additional mana? like...the health we got from golden apple in vanilla minecraft?

Reply:
Also, is "blood pool" same kind like this?

Reply:
Yeah sort of, it interacts a bit more specially but its an additional storage for health/mana effectively.

Reply:
Thanks, for solving my problem, I'm new player that join wynncraft only 1 month XD